core/src/utils/animation/animation-utils.ts

```diff
@@ -1,9 +1,25 @@
+export const getAnimationPrefix = (el: HTMLElement): string => {
+  const supportsUnprefixed = (el.style as any).animationName !== undefined;
+  const supportsWebkitPrefix = (el.style as any).webkitAnimationName !== undefined;
+
+  if (!supportsUnprefixed && supportsWebkitPrefix) {
+    return '-webkit-';
+  }
+
+  return '';
+};
 
 export const setStyleProperty = (element: HTMLElement, propertyName: string, value: string | null) => {
+  if (propertyName.startsWith('animation')) {
+    propertyName = getAnimationPrefix(element) + propertyName;
+  }
   element.style.setProperty(propertyName, value);
 };
 
 export const removeStyleProperty = (element: HTMLElement, propertyName: string) => {
+  if (propertyName.startsWith('animation')) {
```

Comment:
Same thing as my previous comment re: modifying the values of function parameters.